doc/release-notes-16383.md

```diff
@@ -1,8 +0,0 @@
-RPC changes
------------
-
-RPCs which have an `include_watchonly` argument or `includeWatching`
```

Comment:
Looks like this detached release note was lost?

Reply:
this was never fixed right? I don't see it :(